src/main/java/aeneas/models/PieceFactory.java

```diff
@@ -5,11 +5,286 @@
  * @author Joseph Martin
  */
 public class PieceFactory {
-  static Piece[] pieceIndex = {
-    /* TODO: Fill in pieces here */
+  public static Piece[] pieces = {
```

Comment:
Also, make sure it does a deep copy (just using the default implementation of .clone() will perform a shallow copy).